pkgs/development/python-modules/gast/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPythonPackage, fetchurl, lib }:
+
+let
+  pname = "gast";
+  version = "0.2.0";
+in buildPythonPackage rec {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/serge-sans-paille/gast/archive/master.zip";
```

Comment:
fetching from master is a bad idea, not reproducible

Reply:
Yeah, but it doesn't have any release or tag. Should I go for a commit?

Reply:
Yep. Do try to find the commit corresponding to the version.